packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/EnterAmount/index.tsx

```diff
@@ -148,6 +172,7 @@ const EnterAmountContainer = (props: Props) => {
           crossBorderLimits={crossBorderLimits}
           formErrors={formErrors}
           formActions={props.formActions}
+          onMaxButtonClicked={handleMaxButtonClicked}
```

Comment:
Given that there are so many params to the enter amount screen, and also that the analytics call is pretty static, I think it would be better (and a lot less code for you) if we just move the `handleMaxButtonClicked` method inside of the enter amount component.

Reply:
I disagree because in that case I have to pass 2 more props to `EnterAmount` because it doesn't have clue about entered amount nor withdrawal method :/

Reply:
Yeah I see what you mean. Good call!